alerts/nginx/low-request-rate.json

```diff
@@ -0,0 +1,37 @@
+{
+    "displayName": "Nginx - low request rate",
+    "documentation": {
+        "content": "The request rate is derived from the requests metrics taken as a rate of every 5 minutes. This value should be monitored beforehand to understand what qualifies as a normal request rate so a threshold can be established. When request rate is below this threshold, then that means there is an environment problem that are limiting the request rates.",
+        "mimeType": "text/markdown"
+      },
+    "userLabels": {},
+    "conditions": [
+      {
+        "displayName": "VM Instance - workload/nginx.requests",
+        "conditionThreshold": {
+          "aggregations": [
+            {
+              "alignmentPeriod": "300s",
+              "perSeriesAligner": "ALIGN_RATE"
+            }
+          ],
+          "comparison": "COMPARISON_LT",
+          "duration": "0s",
+          "filter": "resource.type = \"gce_instance\" AND metric.type = \"workload.googleapis.com/nginx.requests\"",
+          "thresholdValue": 10,
+          "trigger": {
+            "count": 1
+          }
+        }
+      }
+    ],
+    "alertStrategy": {
+      "autoClose": "604800s",
+      "notificationPrompts": [
```

Comment:
Property "notificationPrompts" does not exist in object specification.

Reply:
updated